change/office-ui-fabric-react-2019-08-07-15-55-26-t-mashia-add_ariaLabelForToggleAllGroupsButton_cherrypick.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "patch",
```

This should be `patch` since we are no longer modifying the public API. My oversight on the original PR, but its OK.